routes/my_stuff.js

```diff
@@ -48,7 +48,7 @@ function getAllDevices(req, engine) {
                         isOnlineAccount,
                         isDataSource,
                         isThingEngine]) => {
-                    return { uniqueId: uniqueId, name: name || req._("Unknown device"),
+                    return { uniqueId: uniqueId, name: name || req._("Unknown device") + ` (${uniqueId})`,
```

Comment:
You should not concatenate translatable strings, because it makes the result hard to translate (see https://www.gnu.org/software/gettext/manual/html_node/Preparing-Strings.html#Preparing-Strings for more info)